client/competitionRegistration.js

```diff
@@ -111,7 +111,7 @@ Template.competitionRegistration.helpers({
 
   registrationAskAboutGuests: function() {
     var competitionId = this.competitionId;
-    competition = Competitions.findOne({ _id: competitionId }, { fields: { registrationAskAboutGuests: 1 } });
+    var competition = Competitions.findOne({ _id: competitionId }, { fields: { registrationAskAboutGuests: 1 } });
```

Comment:
Hmm... that's odd. We have tons of these. See https://github.com/cubing/ccm/blob/master/both/collections/group.js#L1 for example.

Perhaps RubyMine figures that if we're doing this at the outmost scope, we know what we're doing, but if it's inside of a function, it's going to catch us?